lib/SILOptimizer/Mandatory/PMOMemoryUseCollector.cpp

```diff
@@ -11,22 +11,22 @@
 //===----------------------------------------------------------------------===//
```

Comment:
How about a file comment explaining what PMO means, and how this differs from DIMemoryUseCollectorOwnership. Are you going to rename DIMemoryUseCollectorOwnership to DIMemoryUseCollector now? Otherwise, it looks like this is all reformatting. Seems fine to me.

Reply:
Yes. I can add a comment calling out that this PMO means Predictable Memory Operations. The main thing I am trying to do is finish that transition so I am going to rename do DIMemoryUseCollector.

Reply:
I am going to do that in a follow up commit.